jetty-documentation/src/main/asciidoc/development/embedding/embedded-examples.adoc

```diff
@@ -37,7 +37,7 @@ include::examples/embedded-one-webapp.adoc[]
 ==== Web Application with JSP
 
 This example is very similar to the one in the previous section, although it enables the embedded webapp to use JSPs.
-As of jetty-9.2, we use the JSP engine from Apache, which relies on a Servlet Specification 3.1 style ServletContainerInitializer to initialize itself.
+As of jetty-9.2, we use the JSP engine from Apache, which relies on a Servlet Specification 3.1 style `ServletContainerInitializer` to initialize itself.
```

Comment:
Eventually we should get rid of language like "as of jetty-9.2" as we have versioned documentation now.   But OK to retain for now... just noting.